Sources/BraintreeVisaCheckout/BTConfiguration+VisaCheckout.swift

```diff
@@ -0,0 +1,52 @@
+import VisaCheckoutSDK
+
+#if canImport(BraintreeCore)
+import BraintreeCore
+#endif
+
+/// Contains the remote Visa Checkout configuration for the Braintree SDK.
+extension BTConfiguration {
+
+    /// Returns the environment used to run the Visa Checkout SDK.
+    var visaCheckoutEnvironment: String? {
+        json?["environment"].asString()
+    }
+
+    /// Determines if the Visa Checkout flow is available to be used. This can be used to determine if UI components should be shown or hidden.
+    var isVisaCheckoutEnabled: Bool {
+        json?["visaCheckout"]["apiKey"].isTrue ?? false
+    }
+
+    /// The Visa Checkout API Key associated with this merchant's Visa Checkout configuration.
+    var visaCheckoutAPIKey: String? {
+        json?["visaCheckout"]["apiKey"].asString()
+    }
+
+    /// Returns the Visa Checkout External Client ID configured in the Braintree Control Panel
+    var visaCheckoutExternalClientID: String? {
+        json?["visaCheckout"]["externalClientId"].asString()
+    }
+
+    /// The accepted card brands for Visa Checkout.
+    var acceptedCardBrands: [String] {
+        guard let supportedCardTypes = json?["visaCheckout"]["supportedCardTypes"].asStringArray() else {
+            return []
+        }
+
+        return supportedCardTypesToAcceptedCardBrands(supportedCardTypes)
+    }
+
+    /// Returns the supported card types for Visa Checkout to accepted card brands.
```

Comment:
nit: take it or leave it - can we tweak the comment to say:
```suggestion
    /// Returns the accepted card brands for the corresponding Visa Checkout supported card types.
```